### ISSUE TITLE
TwoColumnPanel: content overlap at 200% zoom

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#786]
https://cauldron.dequelabs.com/components/TwoColumnPanel

<img width="517" alt="two column panel component's content overlapping with other content at 200% zoom" src="https://user-images.githubusercontent.com/3180185/191329693-4dad32b6-5dd1-4437-a302-098cc0984125.png">

**Comment by scurker** [comment]
This is partially fixed by #897, we need to adjust to allow for the header not not have a fixed height when the text in the column wraps. This should be taken into consideration when #889 is addressed.
